### PR TITLE
Fix test jest

### PR DESCRIPTION
**Pull request by koooge**
Hi. Upgrade jest and fix ignore pattern.

```
$ npm run test

> frisby@2.1.1 test /path/to/frisby
> jest

No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In /path/to/frisby
  17 files checked.
  testMatch: **/__tests__/**/*.[jt]s?(x), **/?(*.)+(spec|test).[tj]s?(x) - 10 matches
  testPathIgnorePatterns: /__tests__/fixtures/, /node_modules/, /src/ - 0 matches
  testRegex:  - 0 matches
Pattern:  - 0 matches
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frisby@2.1.1 test: `jest`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the frisby@2.1.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/koooge/.npm/_logs/2019-04-02T06_03_18_838Z-debug.log
```

  I could test frisby by `npm test` on my env. `/src/` seems above error, so I replaced with `/src/frisby/spec.js`.

### CODE REVIEW

**Comment by H1Gdev** [comment]
@koooge 

In my environments(linux and windows), even if jest is upgraded, tests will execute correctly.
I think that jest keeps compatibility with testPathIgnorePatterns.

**Comment by koooge** [comment]
I tried this on mac and ubuntu18(and node6, 8, 10), but both of them show the same error (Pattern: - 0 matches). 🤔 why..

**Comment by koooge** [comment]
Aha, I found out my env `~/go/src/github.com/vlucas/frisby` includes `/src/`. That's why jest ignores all tests. So add <rootDir> to ignore pattern/

https://jestjs.io/docs/en/configuration#testpathignorepatterns-array-string

File: package.json [hunk 1]
```diff
@@ -33,15 +33,15 @@
...
   "devDependencies": {
-    "jest": "^21.2.1",
-    "nock": "^9.3.3",
-    "eslint": "^4.19.1"
+    "eslint": "^4.19.1",
+    "jest": "^24.6.0",
+    "nock": "^9.3.3"
   },
   "jest": {
     "testPathIgnorePatterns": [
```
Comment (H1Gdev) [inline package.json]: Do not change order in testPathIgnorePatterns.

**Comment by H1Gdev** [comment]
@koooge 

So, [this commit](https://github.com/koooge/frisby/tree/102142f7c8b66d0da8fe3e3e6d7b349b074a61cb) is unnecessary.

File: package.json (hunk 1 above)
Comment (koooge) [reply package.json]: I fixed it

**Pull request merged**